frontend-react/src/styles/_components.scss

```diff
@@ -79,4 +79,11 @@ hr {
         border-color: color('primary');
         border-width: medium;
     }
+}
+
+.manage-public-key {
+    .usa-icon {
+        height: 1.25em;
```

Comment:
I recommend that we use a consistent unit for sizes/spacings.  USWDS uses 8px as [the atomic size for spacing](https://designsystem.digital.gov/design-tokens/spacing-units/), so we could use pixels for more explicit styling OR use rems if needed.